stdlib/public/core/Arrays.swift.gyb

```diff
@@ -1412,6 +1416,7 @@ extension Array {
   ///
   /// - Complexity: O(`self.count`) if the array is bridged,
   ///   otherwise O(1).
+  @discardableResult
   public mutating func popLast() -> Element? {
```

Comment:
on Sun Apr 03 2016, Chris Lattner <notifications-AT-github.com> wrote:

> In stdlib/public/core/Arrays.swift.gyb:
>
>> @@ -1412,6 +1416,7 @@ extension Array {
>>    ///
>>    /// - Complexity: O(`self.count`) if the array is bridged,
>>    ///   otherwise O(1).
>> +  @discardableResult
>>    public mutating func popLast() -> Element? {
>
> Good point, I'd appreciate @dabrahams thoughts on that.

On second thought, I think we should say, “use removeLast if you don't
want the result.”

-- 
Dave